index.html

```diff
@@ -25,95 +25,333 @@
+                <img class="work-img" src="./project.jpg" alt="project image" />
+                <div class="work-content">
+                  <h4 class="work-item-title">Multi-Post Stories</h4>
+                  <p class="project-section">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
+                  </p>
+                  <ul class="skillset">
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                  <div class="work-item-actions">
+                    <button class="button button-primary">See Project</button>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="divider-full"></div>
+        </section>
+        <section class="contact">
+          <p class="contact-title desktop">Let's chat!</p>
+          <h5 class="contact-title mobile">CONTACT US</h5>
+          <h6 class="contact-description">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </h6>
+          <form
+            class="contact-form"
+            action="https://formspree.io/f/xvolvllr"
+            method="POST"
+          >
+            <input
+              type="text"
+              id="fullname"
+              name="fullname"
+              placeholder="Enter Full Name"
+              maxlength="30"
+              required
+            />
+            <div id="names">
+              <input
+                type="text"
+                id="fname"
+                name="firstname"
+                placeholder="Enter First Name"
+                maxlength="30"
+                required
+              />
+              <input
+                type="text"
+                id="lname"
+                name="lastname"
+                placeholder="Last Name"
+                maxlength="30"
+                required
+              />
+            </div>
+            <input
+              type="email"
+              id="email"
+              name="Email"
+              placeholder="Email Address"
+              required
+            />
+
+            <textarea
+              id="msg"
+              name="user_message"
+              placeholder="Your message"
+              rows="10"
+              cols="50"
+              required
+              minlength="1"
+              maxlength="500"
+            ></textarea>
+
+            <button type="submit" class="send-button">Send</button>
+          </form>
+        </section>
```

Comment:
- [ ] Kindly your mobile version still looks very good, just one thing you have to work on is adding some padding from left and right on the contact section to match the Figma design.

Figma design
![image](https://user-images.githubusercontent.com/35535130/166838603-103994a0-a6fb-44fa-a255-08a39d80a194.png)

Your design
![image](https://user-images.githubusercontent.com/35535130/166838549-9112d06f-2db6-4d0e-9932-9fa03425f910.png)